src/twisted/runner/test/test_procmon.py

```diff
@@ -503,6 +538,22 @@ def test_stopService(self):
         self.assertEqual({}, self.pm.protocols)
 
 
+    def test_restartAllRestartsOneProcess(self):
+        """
+        L{ProcessMonitor.restartAll} succeeds when there are no processes
```

Comment:
I am confused by this test. in the test name there is **One** process, but the docstring talks about **no** processes and there is no full stop :)